### ISSUE TITLE
False negative when multiple `*args` or `**kwargs` are used in callable with ParamSpec

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#9319]
The following example should result in a type error:

```python
from typing import Callable, ParamSpec

P = ParamSpec("P")

def run(func: Callable[P, None], *args: P.args, **kwargs: P.kwargs) -> None:
    func(*args, *args, **kwargs)
    func(*args, **kwargs, **kwargs)
```

This bug also exists in mypy and was reported [here](https://github.com/python/mypy/issues/18035). 

We may want to add this test to the [typing spec conformance test suite](https://github.com/python/typing/blob/main/conformance/tests/generics_paramspec_components.py).

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.387.
